templates/resources/vmaas_load_balancer_monitor.md.tmpl

```diff
@@ -0,0 +1,76 @@
+---
+layout: ""
+page_title: "hpegl_vmaas_load_balancer_monitor Resource - vmaas-terraform-resources"
+subcategory: {{ $arr := split .Name "_" }}"{{ index $arr 1 }}"
+description: |-
+  {{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# Resource hpegl_vmaas_load_balancer_monitor
+
+-> Compatible version >= 5.4.6
+
+{{ .Description | trimspace }}
+`hpegl_vmaas_load_balancer_monitor` resource supports NSX-T Load balancer Monitor creation.
+
+For creating an NSX-T Load balancer Monitor, use the following examples.
+
+-> NSX-T Monitor having HTTP,HTTPS,ICMP,Passive,TCP,UDP  considered as different monitor types. You can create either HTTP,HTTPS,ICMP,Passive,TCP,UDP, but not everything at the same time.
+
+-> Load Balancer ID Data Source `hpegl_vmaas_load_balancer` is supported for Creating Monitor.
+
+
+## Example usage for creating NSX-T Load balancer Monitor for HTTP with all possible attributes
+
+-> For NSX-T Load balancer Monitor for HTTP creation, attribute monitorType value `LBHttpMonitorProfile` is applicable only HTTP creation.
+
+-> Once NSX-T Load balancer Monitor for HTTP created with `LBHttpMonitorProfile`, we can not modify the attribute during the update operation and its disabled from UI.
+
+{{tffile "examples/resources/hpegl_vmaas_load_balancer_monitor/nsx_t_lb_http_monitor.tf"}}
+
+
+## Example usage for creating NSX-T Load balancer Monitor for HTTPS with all possible attributes
+
+-> For NSX-T Load balancer Monitor for HTTPS creation, attribute monitorType value `LBHttpsMonitorProfile` is applicable only HTTPS creation.
```

Comment:
ditto, may not need them. Sync with Jinu once.

Reply:
removed